pkg/jaeger/jaeger.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2019-2020 AccelByte Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package jaeger
+
+import (
+	"context"
+
+	"github.com/emicklei/go-restful"
+)
+
+const (
+	traceIDKey     = "X-Ab-TraceID"
```

Comment:
we have one in logger/event package, can we use that one instead of creating new one here?

Reply:
agree